chore: Disable `forget password` in Prototype + set up a dev account

yanness168 opened the pull request:
### Motivation and context
`forget-password` won't work without email verification, and so we need to disable it when `PROTOTYPE=TRUE`. I find it very annoying that I need to register a new account every time I start the app. Therefore, I created a dev account with shared credentials.

### Changes performed
- Duo Bypass in Prototype Mode
- Allowed Login with Pending Email Verification in Prototype mode because we can't support DUO yet
- Fixed New User Signup Status: before newly registered users in prototype mode were incorrectly created with their `emailVerification` status set to `'verified'` in the database.
- Added a dev account in db
- Disable `forget-password`  by issuing a warning in Prototype mode

### Before submitting this PR, please make sure:
 - [x] Your code builds cleanly without any errors
 - [ ] You have added unit tests, if this PR introduces new functionality
 - [ ] You have added regression tests, if this PR fixes a bug, or raised a task for this
 - [ ] You have updated any related documentation, i.e. README, playbook, or external documentation, or raised tasks for this

Manjinder9 commented on server/src/routes/auth.routes.js:
It looks like in prototype mode we now let users with emailVerification = "pending" log in.
The idea was that only verified users can log in in any mode, and in prototype we just auto mark them as verified instead of sending the email.

we can do something similar like you did in forget password, something like "in prototype bypassing email verification" step and set emailVerification = verified this way the logic remain same in prototype true or false.

<img width="844" height="754" alt="image" src="https://github.com/user-attachments/assets/cebd4188-27bc-4124-b893-4ecdf2552e01" />

yanness168 replied:
But it is indeed the user didn't have `emailVerification` done in PROTOTYPE= true, unless you wrote a `force-pass` endpoint for email verification like KYC has?

I do understand the approach you are suggesting. Let's see what the others think :-)

FTP-droid commented:
> @FTP-droid @Manjinder9 I addressed the comments, and the `clientId not defined server errorr` should be gone.

Thanks, the prototype works without any env setup now. However some duo tests are failing.

yanness168 commented:
> > @FTP-droid @Manjinder9 I addressed the comments, and the `clientId not defined server errorr` should be gone.
> 
> Thanks, the prototype works without any env setup now. However some duo tests are failing.

Sure, will look into it soon

yanness168 commented:
@FTP-droid pls review again :)
